DockerfilePub

```diff
@@ -0,0 +1,44 @@
+# Quickstart: Compose and Rails https://docs.docker.com/compose/rails/
+
+# Base image
+FROM ruby:2.5
+
+# Install dependencies
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
+  apt-get update -qq && apt-get install -y nodejs postgresql-client imagemagick cron
+
+# Add app user
+RUN useradd --create-home appuser
+WORKDIR /home/appuser
+
+# Update permissions
+COPY change_id.sh /root/change_id.sh
+RUN chmod 755 /root/change_id.sh && \
+  /root/change_id.sh -u 194
+
+COPY Gemfile /home/appuser/Gemfile
+COPY Gemfile.lock /home/appuser/Gemfile.lock
+RUN bundle install
+
+# Copy code
+COPY --chown=appuser . /home/appuser
+
+# Guarantees umask is set properly to alleviate issue with umask not sticking inside the node container
+# This is to ensure permissions of files stored on the server will be given the correct permissions
+RUN chmod 0755 /home/appuser/entrypoint.sh && \
+  echo 'umask 002' >> /home/appuser/.profile && \
+  echo 'umask 002' >> /home/appuser/.bashrc
+
+RUN npm install -g bower && \
+  rake bower:install['--allow-root']
+
+USER appuser
+
+RUN rake crt:cron_task:setup_crontab
+
+EXPOSE 8199
```

Comment:
I'd like a comment here on why this is exposed - is this related to delayed jobs?